doc/cookbook/jwt-and-basic-auth/jwt-and-basic-auth.cabal

```diff
@@ -22,7 +22,7 @@ executable cookbook-jwt-and-basic-auth
                      , servant
                      , servant-client
                      , servant-server
-                     , servant-auth ==0.3.*
+                     , servant-auth
```

Comment:
I think we'll want to keep the version bound until we move `servant-auth` in the main repository. Could you try with a higher version ? I am guessing that `>= 0.4.0.0" should work.

Reply:
Obviously, that is why I've put this PR back to `draft`.